test/js_test.go

```diff
@@ -2343,15 +2378,58 @@ func TestJetStreamImportDirectOnly(t *testing.T) {
 		}
 	}
 
-	// Cannot subscribe with JS context from another account right now.
-	if _, err := js.SubscribeSync("ORDERS"); err != nats.ErrJetStreamNotEnabled {
-		t.Fatalf("Expected an error of '%v', got '%v'", nats.ErrJetStreamNotEnabled, err)
+	// Can attach to the consumer from another JS account if there is a durable name.
+	sub, err = js.SubscribeSync("ORDERS", nats.Durable("d4"), nats.BindStream("ORDERS"))
+	if err != nil {
+		t.Fatalf("Unexpected error: %v", err)
+	}
+	waitForPending(t, toSend)
+
+	// Bind has the same effect as above since it would not attempt to create and lookup will work.
+	sub, err = js.SubscribeSync("ORDERS", nats.Bind("ORDERS", "d4"))
+	if err != nil {
+		t.Fatalf("Unexpected error: %v", err)
+	}
+
+	// Even if there are no permissions or import to check that a consumer exists,
+	// it is still possible to bind subscription to it.
+	sub, err = js.PullSubscribe("ORDERS", "d1", nats.Bind("ORDERS", "d1"))
```

Comment:
This does look a bit funny, but might be ok.

I think it might be better to say that nats.Bind only applicable to  normal Subscribe() calls?

Reply:
But if we don't use `nats.Bind()` here then `PullSubscribe` it would attempt to create the consumer

Reply:
I meant you just do a 

```go
js.Subscribe("ORDERS", nats.Bind("MYSTREAM", "d1"))
```

The bound consumer is a pull subscriber already, but maybe that is not the best way to approach.

Reply:
Ah ok I see now, then maybe based on the consumer info lookup would make the client either a push or pull subscriber.